ddtrace/opentrace/scope_manager.py

```diff
@@ -0,0 +1,17 @@
+from opentracing import ScopeManager as OpenTracingScopeManager
+
+
+class ScopeManager(OpenTracingScopeManager):
+    """"""
+
+    def __init__(self):
+        pass
+
+    def activate(self, span, finish_on_close):
+        """"""
+        pass
+
+    @property
+    def active(self):
```

Comment:
so sad that in our implementation it's not a `property`. Otherwise our interface was correct. We can iterate over it once we're going to implement the `ScopeManager` API.